zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -8449,7 +8449,7 @@ components:
         filter:
           description: The job search filters.
           allOf:
-            - $ref: "#/components/schemas/JobFilter"
+            - $ref: "#/components/schemas/JobFilterRequest"
```

Comment:
🔧 It should stay `JobFilter` if possible. The `Request` suffix was mostly removed with 8.8 when we dropped keys as numbers (those were the old requests with number keys, the string key filter DTOs dropped the suffix).